src/lib/one-graph/client.js

```diff
@@ -0,0 +1,835 @@
+const os = require('os')
+
+const dotProp = require('dot-prop')
+const { buildClientSchema, parse } = require('graphql')
+const fetch = require('node-fetch')
+
+const { NETLIFYDEVLOG, NETLIFYDEVWARN, chalk, log } = require('../../utils')
+
+const {
+  defaultExampleOperationsDoc,
+  extractFunctionsFromOperationDoc,
+  generateFunctionsFile,
+  generateHandler,
+  readGraphQLOperationsSourceFile,
+  writeGraphQLOperationsSourceFile,
+  writeGraphQLSchemaFile,
+} = require('./netlify-graph')
+
+const ONEDASH_APP_ID = '0b066ba6-ed39-4db8-a497-ba0be34d5b2a'
+
+const httpOkLow = 200
+const httpOkHigh = 299
+
+const basicPost = async (url, options) => {
+  const reqBody = options.body || ''
+  const userHeaders = options.headers || {}
+
+  const headers = {
+    ...userHeaders,
+    'Content-Type': 'application/json',
+    'Content-Length': reqBody.length,
+  }
+
+  const timeoutMilliseconds = 30_000
+
+  const resp = await fetch(url, {
+    method: 'POST',
+    headers,
+    timeout: timeoutMilliseconds,
+    compress: true,
+    body: reqBody,
+  })
+
+  const respBody = await resp.text()
+
+  if (resp.status < httpOkLow || resp.status > httpOkHigh) {
+    console.warn('Response:', respBody)
+    throw new Error(`Netlify OneGraph return non - OK HTTP status code: ${resp.status}`)
```

Comment:
```suggestion
    throw new Error(`Netlify Graph returned invalid HTTP status code: ${resp.status}`)
```
This is a bit easier to read